### PR TITLE
Showing PR flow from May

### LINKED ISSUE #17: Add git file bio
Add a new file to GIT

### PR DESCRIPTION
**Pull request by maytran**
This fixes #17.

-[] task 1
-[] task2

### CODE REVIEW

**Comment by maytran** [comment]
@loriNakamura Would you help me get the image during lunch break?   :heart_eyes:

File: maytran-bio.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+##  Header
+This is just an example of a bio file.
+
+## Name: May Tran
+## Section: 393
+## Phone: 4-2219
+
+## Project support
+All JPL projects.
```
Comment (maytran) [inline maytran-bio.md]: Are you sure it's All JPL projects?

Comment (maytran) [inline maytran-bio.md]: Really?

Comment (maytran) [inline maytran-bio.md]: R u sure?

Comment (maytran) [inline maytran-bio.md]: @loriNakamura Please take a look.

**Comment by loriNakamura** [comment]
@maytran Hi, May.  Please add your Group letter in 393.  Thanks!  Lori :elephant:

**Comment by maytran** [comment]
@loriNakamura Please take a look.

File: maytran-bio.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+## Favorite project
+Cassini
```
Comment (maytran) [inline maytran-bio.md]: @loriNakamura Just added Cassini.

**Comment by loriNakamura** [comment]
Looks good -- thanks!  :+1:

File: maytran-bio.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+##  Header
+This is just an example of a bio file.
+
+## Name: May Tran
+## Section: 393B
```
Comment (maytran) [inline maytran-bio.md]: @loriNakamura Just added letter to group.

**Comment by loriNakamura** [comment]
@maytran Thanks!  :octocat:

**Pull request merged**